gno.land/pkg/gnoweb/app.go

```diff
@@ -31,28 +31,29 @@ type AppConfig struct {
 	ChainID string
 	// AssetsPath is the base path to the gnoweb assets.
 	AssetsPath string
-	// AssetDir, if set, will be used for assets instead of the embedded public directory
+	// AssetDir, if set, will be used for assets instead of the embedded public directory.
 	AssetsDir string
 	// FaucetURL, if specified, will be the URL to which `/faucet` redirects.
 	FaucetURL string
+	// Domain is the domain used by the node.
+	Domain string
 }
 
 // NewDefaultAppConfig returns a new default [AppConfig]. The default sets
 // 127.0.0.1:26657 as the remote node, "dev" as the chain ID and sets up Assets
 // to be served on /public/.
 func NewDefaultAppConfig() *AppConfig {
 	const defaultRemote = "127.0.0.1:26657"
-
 	return &AppConfig{
-		// same as Remote by default
 		NodeRemote: defaultRemote,
 		RemoteHelp: defaultRemote,
 		ChainID:    "dev",
 		AssetsPath: "/public/",
+		Domain:     "gno.land",
```

Comment:
Didn't we discuss this getting this from the remote genesis.json, together with the chain id?

Reply:
It should ideally be queried from the running node.

Reply:
I definitely want to do it. However, I didn't change any real behaviors (or really minimally, like some errors) in this PR. It's mostly a cleanup iteration with documentation and tests. So i prefer keeping this for a next PR.